ignite/pkg/availableport/availableport_test.go

```diff
@@ -0,0 +1,83 @@
+package availableport_test
+
+import (
+	"fmt"
+	"math/rand"
+	"testing"
+
+	"github.com/ignite/cli/ignite/pkg/availableport"
+	"github.com/stretchr/testify/require"
+)
+
+func TestAvailablePort(t *testing.T) {
```

Comment:
I think you can write all the test cases within `TestAvailablePort` using Go table driven tests.

You can find an example in [ignite/pkg/xurl/xurl_test.go](https://github.com/ignite/cli/blob/6cb985b5523b34d29dd99c05b15c72c10ea838a3/ignite/pkg/xurl/xurl_test.go#L9), you can also find more examples within other test files.